CMakeLists.txt

```diff
@@ -45,11 +45,11 @@ include(${CMAKE_CURRENT_BINARY_DIR}/cmake/CPM.cmake)
 
 ##################################  MaCh3  ######################################
 if(NOT DEFINED MaCh3_Branch)
-  SET(MaCh3_Branch "develop")
+  SET(MaCh3_Branch "feature_core_1.2.0")
 endif()
 
 #If MaCh3 was sourced find it, otherwise use CPM
-#find_package(MaCh3 QUIET)
+find_package(MaCh3 QUIET)
```

Comment:
can you fix this?

Reply:
Can you leave commneted out for the moment? This is bit hack. But currnetly we call we call Core valdiations in MaCh3 contaiiners. Sometimes it was finidng MaCh3 from contianer instead of feature branch